src/shortcuts.js

```diff
@@ -436,7 +436,7 @@ export const globalShortcuts = perma(() => [
     keyboard: { key: 'f', shift: true, meta: true },
     exec: () => {
       const state = store.getState()
-      store.dispatch({ type: 'search', value: state.search == null ? '' : null })
+      store.dispatch({ type: 'search', value: state.search || window.getSelection().toString() })
```

Comment:
Notice the `state.search == null` check, which is equivalent to `state.search === null || state.search === undefined`. You have accidentally broken the toggle behavior. Try this:

```
state.search == null ? window.getSelection().toString() : null
```